### ISSUE TITLE
RReLU doc doesn't specify the eval mode behaving just like LeakyReLU

### ISSUE THREAD

**Issue by titaiwangms** [pytorch/pytorch#82677]
### 📚 The doc issue

[RReLU](https://pytorch.org/docs/stable/generated/torch.nn.RReLU.html) behaves the same as LeakyReLU when it's on eval mode, but the documentation doesn't seem to provide such information.

### Suggest a potential alternative/fix

Maybe add a link or a notice specifying this would be better.

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

**Comment by jbschlosser** [comment]
Agreed this would be nice to have :) We'd accept a PR describing this, including how the slope for `LeakyReLU` behavior is calculated.
